network/src/commonMain/kotlin/com/wire/kalium/network/tools/KtxSerializer.kt

```diff
@@ -12,5 +12,9 @@ object KtxSerializer {
         // explicitNulls, defines whether null property
         // values should be included in the serialized JSON string.
         explicitNulls = false
+
+        // If API returns null or unknown values for Enums, we can use default constructor parameter to override it
+        // https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/json.md#coercing-input-values
+        coerceInputValues = true
```

Comment:
as alternative, on network layer we can read all enums as `String` and the Mappers in logic handel String -> enum mapping

Reply:
This emplies that the logic module would know how to "deserialize" the network modules. Also, adds some extra-complication to our code.

Instead of:

```kotlin
// Network part
@Serializable
data class Foo(@SerialName("type") val type: String?)


// Logic part
enum class Type {A, B, C}
// Logic knows a bit too much about how data format
fun fromStringToType(stringValue: String?): Type = when(stringValue){
    "a" -> Type.A
    "b" -> Type.B
    "c" -> Type.C
    else -> Type.A
}
``` 

We can just do:

```kotlin
// Network
@Serializable
data class Foo(val type: Type = Type.A)

@Serializable
enum class Type {
    @SerialName("a") A,
    @SerialName("b") B, 
    @SerialName("c") C
}
```